Code/Framework/AzCore/AzCore/DOM/Backends/JSON/JsonBackend.cpp

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzCore/DOM/Backends/JSON/JsonBackend.h>
+
+#include <AzCore/DOM/DomBackendRegistry.h>
+
+namespace AZ::DOM
+{
+    Visitor::Result JsonBackend::ReadFromStringInPlace(AZStd::string& buffer, Visitor* visitor)
+    {
+        return Json::VisitSerializedJsonInPlace(buffer, visitor);
+    }
+
+    Visitor::Result JsonBackend::ReadFromString(AZStd::string_view buffer, Lifetime lifetime, Visitor* visitor)
+    {
+        return Json::VisitSerializedJson(buffer, lifetime, visitor);
+    }
+
+    AZStd::unique_ptr<Visitor> JsonBackend::CreateStreamWriter(AZ::IO::GenericStream* stream)
+    {
+        return Json::GetJsonStreamWriter(stream, Json::OutputFormatting::PrettyPrintedJson);
+    }
```

Comment:
This is missing an argument to allow the user to change the output formatting.
It assumes they want to always use pretty printed json

Reply:
Fully exposed now (as a template parameter to the backend).